src/app/api/backers/[backer]/tx-history/context/route.ts

```diff
@@ -95,15 +161,34 @@ export async function GET(req: Request, { params }: { params: Promise<{ backer:
         if (limitToBack) qb.whereRaw('FALSE')
       })
 
-    const combinedQuery = allocationHistory.union(claimedRewardsHistory)
+    const combinedQuery = allocationHistory.union(claimedRewardsHistory).as('base')
+
+    const orderByColumn =
```

Comment:
Why do we need to validate `totalAmount` and `totalamount`?

Before we used to had `sortBy || 'blockTimestamp', sortDirection || 'desc'`